config/messaging.artemis.yml

```diff
@@ -0,0 +1,15 @@
+---
+base: &base
+  hostname: localhost
+  port: 61616
+  username: admin
+  password: smartvm
+
+development:
+  <<: *base
+
+production:
+  <<: *base
+
+test: &test
```

Comment:
Why does test have an anchor (also down below)?  I'm guessing this was cargo-culted from database.yml which had it there because of the old metric_fu that was a "copy" of test.

Reply:
is this an infinite loop?
you trying to bomb our build?

Reply:
Yep, this was copied from database.yml, here's a cleanup https://github.com/ManageIQ/manageiq/pull/20056